### PR TITLE
[SKIP-PREVIEW] ROC-3859 JS other fixes

### PR DESCRIPTION
**Pull request by timja-kainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-3859

### Change description
Add / remove JS enhancement

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/views/income-expense-scripts.njk [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<script src="{{ asset_paths['js'] }}/income-expense-calculation.js"></script>
+<script src="{{ asset_paths['js_vendor'] }}/nodelist-foreach-polyfill.js"></script>
+<script src="{{ asset_paths['js_vendor'] }}/classlist-polyfill.js"></script>
+<script src="{{ asset_paths['js_vendor'] }}/startswith-polyfill.js"></script>
```
Comment (singh-surjeet) [inline src/main/views/income-expense-scripts.njk]: Out of interest, are classlist-polyfill.js and startswith-polyfill.js in use?

Comment (timja-kainos) [reply src/main/views/income-expense-scripts.njk]: not currently, i'm reworking this atm

**Comment by geek-soft** [comment]
Shouldn't be that hard. We can list through the list of errors in top box, find links pointing to non existing elements and then we can just remove them. Will have a chat with @ckalista .

**Comment by timja-kainos** [comment]
Cool sounds find, but still out of scope for this PR

**Pull request merged**